### PR TITLE
Flutter DeviceConnect Document

### PR DESCRIPTION
**Pull request by ChiragRayani**

### CODE REVIEW

File: docs/device-connect/flutter.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+::: warning NOTE
+Following will be shared by FinBox team at the time of integration:
+- `ACCESS_KEY`
+- `SECRET_KEY`
+- `DC_SDK_VERSION`
+- `CLIENT_API_KEY`
+:::
+
+## Integration Flow
+Assuming the bridge has been setup between your project and DeviceConnect as per [this](/device-connect/react-native.html#setting-up-the-bridge) section, the following would be the flow in your app:
```
Comment (sanoopsandy) [inline docs/device-connect/flutter.md:13]: @ChiragRayani  Why react native bridge?

File: docs/device-connect/flutter.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+<img src="/client_workflow.png" alt="Client Workflow" style="width:80%;height:80%" />
+
+### Step 1: Requesting Runtime Permissions
+It is required to show what all permissions you will be needing from users in your app, and then ask them for the permissions. Please refer [this](/device-connect/react-native.html#handle-permissions) section to get the list of permissions the SDK needs. Also in case you want to exclude certain permissions, you can use a `remove` rule as mentioned in the same article.
```
Comment (sanoopsandy) [inline docs/device-connect/flutter.md:18]: @ChiragRayani Here also react native ka reference hai

File: docs/device-connect/flutter.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+::: tip TIP
+- It is recommended that `CUSTOMER_ID` is a masked value not a unique personal identifier like a phone number or email id so that the user remains anonymous to FinBox.
+- SDK will automatically consider syncing based on whether permission was granted by the user or not and what was configured, hence the `createUser` method must be called even though the user denies certain permissions.
+:::
+
+`createUser` in general acts as a check for API credentials. For the first time when the user doesn't exists, it will create a user on the FinBox side. The next step will work only if this function returns a success response.
+
+### Step 3: Start Syncing Data
+If the `createUser` response is successful, you can call `startPeriodicSync` function (Refer [this](/device-connect/react-native.html#start-periodic-sync-method) article) which will sync data in period intervals in background.
```
Comment (sanoopsandy) [inline docs/device-connect/flutter.md:31]: Here also

**Review by sanoopsandy — COMMENTED** [review]
Remove React native references.

File: docs/device-connect/flutter.md (hunk 1 above)
Comment (ChiragRayani) [reply docs/device-connect/flutter.md]: done

File: docs/device-connect/flutter.md (hunk 2 above)
Comment (ChiragRayani) [reply docs/device-connect/flutter.md]: fix

File: docs/device-connect/flutter.md (hunk 3 above)
Comment (ChiragRayani) [reply docs/device-connect/flutter.md]: fix

**Pull request merged**